hack/build.sh

```diff
@@ -4,6 +4,14 @@ set -ex
 
 cd "$(dirname "$0")/.."
 
+REAL_GOPATH=$(realpath "${GOPATH:-GOPATH-NOT-SET}")
```

Comment:
This assumes a single path in `GOPATH` (it can contain multiple, colon-delimited paths).  Maybe:

```diff
diff --git a/hack/build.sh b/hack/build.sh
index bfd3d01..27be95e 100755
--- a/hack/build.sh
+++ b/hack/build.sh
@@ -2,8 +2,16 @@
 
 set -ex
 
+LAUNCH_PATH="${PWD}"
 cd "$(dirname "$0")/.."
 
+PACKAGE_PATH="$(go list -e -f '{{.Dir}}' github.com/openshift/installer)"
+LOCAL_PATH="${PWD}"
+if test "${PACKAGE_PATH}" != "${LOCAL_PATH}" ; then
+    echo "build from your \${GOPATH} (${PACKAGE_PATH}, not ${LAUNCH_PATH}" 2>&1
+    exit 1
+fi
+
 MODE="${MODE:-release}"
 LDFLAGS="${LDFLAGS} -X main.version=$(git describe --always --abbrev=40 --dirty)"
 TAGS="${TAGS:-}"
```

Reply:
Thanks for the fixup!  Works for me.  I'll push the update here.